hadoop-hdds/common/pom.xml

```diff
@@ -226,6 +226,10 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <groupId>org.apache.ozone</groupId>
       <artifactId>hdds-interface-admin</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.ozone</groupId>
+      <artifactId>hdds-interface-server</artifactId>
+    </dependency>
```

Comment:
@Xushaohong , which class reference requires this new dependency?

Reply:
Originally it is for DeletedBlocksTransaction in **DeletedBlocksTransactionInfoWrapper**,  I have now moved the wrapper class  to a better place.